examples/model_assisted_labeling/tiled_imagery_mal.ipynb

```diff
@@ -19,18 +19,12 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    geo_polygons = [transformer(point) for point in points]
</pre><p>Same feedback here. The projection of a shapely polygon should be thoughtless.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>